tests/end2end/cypress/integration/requests-service-ghaction.js

```diff
@@ -33,6 +33,24 @@ describe('Request service', function () {
             })
     })
 
+    it('WFS GetFeature', function () {
+        cy.request({
+            method: 'POST',
+            url: 'https://demo.snap.lizmap.com/lizmap_3_5/index.php/lizmap/service/?repository=test1&project=selection',
```

Comment:
Is-it wanted ? `demo.snap.lizmap.coom`
It's not a good idea to use a real server for testing.

Reply:
it is not wanted. I have fix it.